tools/crm_mon.c

```diff
@@ -658,7 +657,7 @@ static GOptionEntry deprecated_entries[] = {
  * \return Bitmask of pe_print_options suitable for resource print functions
  */
 static unsigned int
-get_resource_display_options(unsigned int mon_ops)
+get_resource_display_options()
```

Comment:
Explicitly say (void), recent compilers complain about that

Reply:
Fixed.